element-templates/inbound-template-connector.json

```
@@ -45,13 +49,36 @@
       "type": "String",
       "label":"Sender",
       "feel": "optional",
-      "group":"endpoint",
+      "group":"config",
       "description":"Message sender name",
       "value": "Alice",
       "binding": {
         "type": "zeebe:property",
         "name": "sender"
       }
+    },
+    {
+      "label": "Event variable",
```

Let's keep it consistent with other connectors.

```suggestion
      "label": "Result variable",
```

---

Sounds good, works for me :)